src/book/02-system/03-tools/06-natnet.mdx

```diff
@@ -0,0 +1,104 @@
+---
+section: System
+chapter: Tools
+title: NatNet SDK
+description: A tool that provides positional infomation from motive mocap software.
+slug: /system/tools/natnet_sdk
+authors:
+  - Luke Haigh
+---
+
+NatNet is module that works in conjunction with the [OptiTrack Motive software](https://www.optitrack.com/software/motive/) and motion tracking camera system. The purpose of the NatNat system is to record positional information about the NUgus robots in relation to the playing field and the external envrionment so that it can be provided to our automatic anotation system which will use the positional information of the robots to automatically create segmentation masks of the environment. 
```

Comment:
Make this description more generic - also make the distinction between NatNet the networking sdk from Motive and NatNet the module in the NUbots codebase. NatNet is it's own thing, we just have a module to capture the NatNet packets. As for making it more generic, don't mention the automatic annotator, just say in general that it can be used to get motion capture data, and for example could be used to get positional information from the environment (field, robot, etc).

Reply:
Also watch for typos.